CONTRIBUTE.md

```diff
@@ -64,6 +64,7 @@ deploy, and therefore we try to avoid it.
 * Add **unit tests** to verify that your code not only works but also keeps working over time. When adding tests, keep
 the same directory structure used for regular classes. Try to cover **all your code** with tests. The bigger the test
 coverage, the more reliable and better our library is.
+    * `phpunit --configuration tools/phpunit tests/path/to/your/test.php`
```

Comment:
It would be nice to elaborate a bit more.

On the other hand, as long as the test is inside the `tests` directory, there's no need to pass it in the command line, as phpunit will run all tests it finds in there.